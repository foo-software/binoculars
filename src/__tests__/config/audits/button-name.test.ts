import auditButtonName from '../../../config/audits/button-name';

describe('button-name audit', () => {
  it('should match snapshot', () => {
    expect(auditButtonName('en')).toMatchSnapshot();
  });
});
