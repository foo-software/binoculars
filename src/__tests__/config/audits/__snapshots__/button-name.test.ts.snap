// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button-name audit should match snapshot 1`] = `
Object {
  "description": "When a button doesn't have an accessible name, screen readers announce it as \\"button\\", making it unusable for users who rely on screen readers. [Learn more](https://web.dev/button-name/).",
  "title": "Buttons have an accessible name",
}
`;
